Sources/TuistCache/Mappers/Graph/TargetsToCacheBinariesGraphMapper.swift

```diff
@@ -49,22 +49,27 @@ public final class TargetsToCacheBinariesGraphMapper: GraphMapping {
     /// The caching profile.
     private let cacheProfile: TuistGraph.Cache.Profile
 
+    /// List of targets that will not use pre-compiled binaries from the cache.
+    private let excludedTargets: Set<String>
```

Comment:
For consistency with `sources`
```suggestion
    private let excludedSources: Set<String>
```